src/web/View.tsx

```diff
@@ -82,19 +82,17 @@ export class View extends ViewBase<Types.ViewProps, {}> {
         focusManager: PropTypes.object
     };
 
-    private focusManager: FocusManager;
+    private _focusManager: FocusManager;
 
-    private resizeDetectorAnimationFrame: number;
-    private resizeDetectorNodes: { grow?: HTMLElement, shrink?: HTMLElement } = {};
+    private _resizeDetectorAnimationFrame: number;
+    private _resizeDetectorNodes: { grow?: HTMLElement, shrink?: HTMLElement } = {};
 
     constructor(props: Types.ViewProps) {
         super(props);
-        if (this.props.restrictFocusWithin) {
-            this.focusManager = new FocusManager();
-        }
+        this._focusManager = new FocusManager();
```

Comment:
I think that's a reasonable restriction. Let's go back to your previous implementation and document it as a read-only prop.